conversations/api/src/main/java/org/sakaiproject/conversations/api/model/ConversationsTopic.java

```diff
@@ -87,6 +88,7 @@ public class ConversationsTopic implements PersistableEntity<String> {
     @Column(name = "HOW_ACTIVE")
     private Integer howActive = 0;
 
+    @Type(type = "org.hibernate.type.InstantType")
```

Comment:
In our version of hibernate the Type for Instant shouldn't be needed anymore

Reply:
You're right. There's something fishy going on. I'll spend a bit more time on this I think, and check grader and tasks while I'm at it.